### ISSUE TITLE
sim2real training reward

### ISSUE THREAD

**Issue by charlietcheng** [younggyoseo/FastTD3#24]
Hi. Thanks for releasing the great work! I was trying to run the sim2real code over [here](https://github.com/younggyoseo/FastTD3/blob/main/sim2real.md) and notice that the reward is getting stuck around 0. Below are some training curves I got with running the command: `python fast_td3/train.py --env_name T1LowDimJoystickRoughTerrain --exp_name FastTD3 --use_domain_randomization --use_push_randomization --total_timesteps 1000000 --render_interval 0 --seed 2
`

<img width="1652" height="794" alt="Image" src="https://github.com/user-attachments/assets/370b4289-52af-4b09-8c5d-597c34b31059" />

Do you know how to fix this? Thanks in advance for your help!

**Comment by younggyoseo** [comment]
Hmm that's strange, I'll have a look at it. Meanwhile, could you try running it with --no_compile?

**Comment by younggyoseo** [comment]
For future reference, this happens in some random machines when using torch.compile and I really don't know why this happens.. It is kind of resolved if you put `print()` somewhere inside training loop but terminal will become very messy -- though training will be faster with compilation.
